index.html

```diff
@@ -91,43 +95,65 @@ <h2>
         Badge model
       </h2>
       <p>
-        There is a single global badge associated with each <a data-cite=
-        "appmanifest#installable-web-applications">web application</a>. At any time, the application's badge
-        is set to one of the following values:
+        A <dfn>badge</dfn> may be one of the following values:
       </p>
       <ul>
-        <li>The special value <a>nothing</a>.
+        <li>The special value <dfn>nothing</dfn>.
         </li>
-        <li>The special value <a>flag</a>.
+        <li>The special value <dfn>flag</dfn>.
         </li>
         <li>An <code><a data-cite="WEBIDL#idl-unsigned-long-long">unsigned long
         long</a></code> value, which MUST NOT be 0.
         </li>
       </ul>
       <p>
-        The user agent MUST initialize each application's badge to
-        <a>nothing</a>. The user agent MAY reset an application's badge to
-        <a>nothing</a> at its discretion (for example, when the system is
-        restarted).
+        Each <a data-cite="dom#concept-document">document</a> and each
```

Comment:
ProTip: use the `[=term=]` syntax for concepts. That will link to DOM.  
```suggestion
        Each [=document=] and each
```